linux/installation/mde_installer.sh

```diff
@@ -401,7 +401,30 @@ verify_conflicting_applications()
     local conflicting_apps=$(timeout 5m find /proc/*/fdinfo/ -type f -print0 2>/dev/null | xargs -r0 grep -Fl "fanotify mnt_id" 2>/dev/null | xargs -I {} -r sh -c 'cat "$(dirname {})/../cmdline"')
 
     if [ ! -z $conflicting_apps ]; then
-        script_exit "found conflicting applications: [$conflicting_apps], aborting" $ERR_CONFLICTING_APPS
+    
+        if [ $conflicting_apps == "/opt/microsoft/mdatp/sbin/wdavdaemon" ]; then
+	
+                    op=$(command -v mdatp)
+
+                    #make sure mdatp is installed
+                    if [ ! -z $op ]; then
+
+                            #check if mdatp is onboarded or not
+                            is_onboarded=$(mdatp health --field healthy | tail -1)
+                            if [ "$is_onboarded" = false ]; then
```

Comment:
`mdatp health --field healthy` can be false for many reasons. So it would be better to check specific health_issue.

Reply:
handled this case